data/mouse-power-effs.csv

```diff
@@ -1067,4 +1067,18 @@ Paragon of the Lawless,124100,0,0,0,0,0,0,0,0,500,0
 Paragon of Strength,305875,0,0,0,0,0,500,0,0,0,0
 Paragon of Shadow,246650,0,0,0,0,0,0,500,0,0,0
 Paragon of Tactics,246965,0,0,0,0,0,0,0,500,0,0
-Richard the Rich,33725,100,100,100,100,0,100,100,100,100,0
+Richard the Rich,33725,200,200,200,200,0,200,200,200,200,0
+Glamorous Gladiator,28575,0,0,0,0,0,600,0,0,0,0
+Zealous Academic,28000,0,0,0,0,0,0,600,0,0,0
+Rocketeer,28575,0,0,0,0,0,0,0,600,0,0
```

Comment:
Rocketeer and Sky Glider MPs don't match what's shown in this sheet: https://docs.google.com/spreadsheets/d/14dw0_YOnIanEaGkm3ABllDxytghJyyiXOp3FX0be1xw/edit#gid=521526777

Reply:
@w0en

Reply:
Changed and pushed